src/bindings/js/node/tests/unit/basic.test.js

```diff
@@ -5,8 +5,14 @@
 const { addon: ov } = require('../..');
 const assert = require('assert');
 const { describe, it, before, beforeEach } = require('node:test');
-const { testModels, getModelPath, isModelAvailable } = require('./utils.js');
+const {
+  testModels,
+  compareModels,
+  getModelPath,
+  isModelAvailable,
+} = require('./utils.js');
 const epsilon = 0.5;
+const outDir = 'tests/unit/out/';
```

Comment:
Does after unit tests run git has clear state?

Reply:
Good point, I believe I should add `tests/unit/out` to `.gitignore`, similar to how it was done for `tests/unit/test_models` path.

Reply:
Please use [mkdtemp()](https://nodejs.org/api/fs.html#fs_fs_mkdtemp_prefix_options_callback). The prefix can be simple. You may also see [this usage of os.tempdir()](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/node/scripts/download_runtime.js#L90)

Reply:
Adding files to `.gitignore` won't remove files in the CI's check